pkg/rpc/clusters/clusters.twirp.go

```diff
@@ -1,39 +1,32 @@
-// Code generated by protoc-gen-twirp v7.1.1, DO NOT EDIT.
+// Code generated by protoc-gen-twirp v8.1.0, DO NOT EDIT.
```

Comment:
This file should not have changed

Reply:
`protoc-gen-twirp` got updated somehow after I followed the instructions on README.

```
go get -u github.com/golang/protobuf/protoc-gen-go
go get -u github.com/twitchtv/twirp/protoc-gen-twirp
go get -u go.larrymyers.com/protoc-gen-twirp_typescript
```

As we haven't fixed the version of `protoc-gen-twirp` binary.
Every time a contributor follows the instructions on README and does a clean setup, they might end up having a newer set of binaries in their system.

Reply:
Maybe we should fix the README content too?

Reply:
We should probably pin those versions if possible, and use a `tools` pattern for this like `weave-gitops`. In any case, let's remove these changes from version control for this PR and see if we can get a more deterministic set of instructions.